rest-api/etl/etl.sql

```diff
@@ -1783,53 +1781,20 @@ DROP TABLE IF EXISTS cdm.temp_obs;
 -- -------------------------------------------------------------------
 TRUNCATE TABLE cdm.fact_relationship;
 
--- -------------------------------------------------------------------
--- unit: observ.meas[1,2] - to link measurements and their qualifiers
--- -------------------------------------------------------------------
-DROP TABLE IF EXISTS cdm.tmp_fact_rel_om;
-
--- -------------------------------------------------------------------
--- This is temporary table for saving original relationships between
--- measurements and their qualifiers in source tables.
--- -------------------------------------------------------------------
-CREATE TABLE cdm.tmp_fact_rel_om AS 
-SELECT
-    cdm_meas.measurement_id         AS measurement_id,
-    cdm_obs.observation_id          AS observation_id
-FROM cdm.measurement cdm_meas
-INNER JOIN rdr.measurement_to_qualifier  mtq
-    ON cdm_meas.meas_id = mtq.measurement_id
-INNER JOIN cdm.observation cdm_obs
-    ON mtq.qualifier_id = cdm_obs.meas_id
-;
-
 -- -------------------------------------------------------------------
 -- Insert to fact_relationships measurement-to-observation relations
 -- -------------------------------------------------------------------
 INSERT INTO cdm.fact_relationship
 SELECT
     21                              AS domain_concept_id_1,     -- Measurement
-    fr.measurement_id               AS fact_id_1,
+    mtq.measurement_id              AS fact_id_1,
     27                              AS domain_concept_id_2,     -- Observation
-    fr.observation_id               AS fact_id_2,
+    cdm_obs.observation_id          AS fact_id_2,
     581411                          AS relationship_concept_id,  -- Measurement to Observation
     'observ.meas1'                  AS unit_id
-FROM cdm.tmp_fact_rel_om fr
-;
-
--- -------------------------------------------------------------------
--- Insert to fact_relationships backwards observation-to-measurement
--- relations
--- -------------------------------------------------------------------
-INSERT INTO cdm.fact_relationship
-SELECT
-    27                              AS domain_concept_id_1,     -- Observation
-    fr.observation_id               AS fact_id_1,
-    21                              AS domain_concept_id_2,     -- Measurement
-    fr.measurement_id               AS fact_id_2,
-    581410                          AS relationship_concept_id,  -- Observation to Measurement
-    'observ.meas2'                  AS unit_id
-FROM cdm.tmp_fact_rel_om fr
+FROM cdm.observation cdm_obs
+INNER JOIN rdr.measurement_to_qualifier mtq
+    ON mtq.qualifier_id = cdm_obs.meas_id
```

Comment:
Was there no purpose to the previous approach of autogenerated IDs - seems like that was pretty roundabout? For example, is the 'measurement_id' something that could be an identifier which we wouldn't want to expose in the CDR?

Reply:
In the ultimate CDR, it's going to be important to have different IDs from the IDs in the RDR. But there is going to be a merging process of combining the OMOP from the RDR with OMOP from EHR data, which will have to reassign IDs. So there's no need to do it here for that reason.

The only reason to auto-assign IDs in this ETL is to handle cases where there are inserts into the same CDM table from multiple sources, which might conflict. This is the case for the observation table, I believe, but not the measurement table.